tests/integration/config_capture_e2e/nodejs/index.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+// Copyright 2016-2022, Pulumi Corporation.  All rights reserved.
```

Comment:
Nit: no harm in the change, but also didn't change anything here so could revert.